tools/initial_ros_setup.sh

```diff
@@ -74,7 +74,7 @@ download_ros2() {
 }
 
 download_ros2_dependencies() {
-  echo "Downloading the dependencies workspace"
+  echo "Downloading the ROS 2 dependencies workspace"
```

Comment:
Should we be consistent here? We call it ros2_dependencies.repos but download it into navstack_dependencies_ws

Reply:
Yeah. That's been bugging me too. It's an accident of history but I felt like it wasn't worth messing up everybody's workspace to fix it.

Reply:
I changed ros2_ws -> ros2_dependencies_ws to homologate with the other workspace names.

Reply:
Oh - I see what you mean. I still think it's worth having the 2 separate workspaces for ros2 things vs direct upstream dependencies. I think I'll just clarify that those are navigation ROS1 and navigation ROS2 dependencies vs ROS2 itself. See updated.

Reply:
Yes. I agree. I think the two workspaces still makes sense. The problem is that `navstack_dependencies_ws` is badly named. It is left over from when we were working in a different repo and didn't have any ROS 1 dependencies.

It should be called something like `additional_ros2_dependencies` or something. But changing that name will force everyone to reset their environments.

Maybe now is the time to fix the naming, though, since we are going to force everyone to switch the ros2_ws to master instead of bouncy soon anyhow.